### ISSUE TITLE
Misleading `reportImplicitOverride` when deriving from a class with unknown parent

### ISSUE THREAD

**Issue by maflAT** [microsoft/pyright#7560]
**Describe the bug**
When deriving from a class, that itself derives from an unknown parent, every method in that class will be reported as implicit override.

**Code or Screenshots**
```python
class Child(UnknownParent):  # reportUntypedBaseClass
    def foo(self) -> None:  # no error
        pass

class GrandChild(Child):
    def bar(self) -> None:  # reportImplicitOverride
        pass
```
![grafik](https://github.com/microsoft/pyright/assets/13979550/b36e5c81-0ae5-49b7-a878-4b021d1e5002)

I understand that the type checker can't determine if a method is overridden in this case or not, so some error should be reported. But I find the implicit override error for methods of the `GrandChild` class misleading. Especially considering that the same error is not reported for methods of the `Child` class.

I came across this issue with the pyqtgraph package.
It has many classes (GraphicsView, GraphicsWidget, ...) that derive from a Qt base class (like QtWidgets.QWidget) and those base classes are unknown at edit time.

**VS Code extension or command-line**
Pyright 1.1.355 integrated in Pylance 2024.3.101

**Comment by erictraut** [comment]
Yes, I forgot about this case when I implemented the `reportImplicitOverride` check. I agree that it pyright not generate a diagnostic in this case.

This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.356, which I just published.
